lib/commands/test-init.ts

```diff
@@ -30,19 +34,38 @@ class TestInitCommand implements ICommand {
 		}
 
 		const dependencies = this.frameworkDependencies[frameworkToInstall] || [];
-		const modulesToInstall = ['karma', 'karma-' + frameworkToInstall, 'karma-nativescript-launcher'].concat(dependencies.map(f => 'karma-' + f));
+		const modulesToInstall: IDependencyInformation[] = [
+			{
+				name: 'karma',
+				// Hardcode the version unitl https://github.com/karma-runner/karma/issues/3052 is fixed
+				version: "2.0.2"
+			},
+			{
+				name: `karma-${frameworkToInstall}`
+			},
+			{
+				name: 'karma-nativescript-launcher'
+			}
+		];
+
+		modulesToInstall.push(...dependencies.map(f => ({ name: f })));
```

Comment:
Shouldn't be `{name: 'karma-' + f }`?

Reply:
nope, the required names already have karma  - https://github.com/NativeScript/nativescript-cli/pull/3698/files#diff-7a3a35d2e2f3966bf14d55b5968c0257R8